### PR TITLE
Line 13 - New Update Exceptions.txt

### PR DESCRIPTION
**Pull request by mbxIO**
Edit Notes:
1. Exception Repeats - 253485 The Syndicate , 269783 Reckless, 271936 Played
2. Stopped at Line 13

### CODE REVIEW

**Comment by mbxIO** [comment]
@miigotu better this way? and should I stop here for now so you can review or can I continue?

File: exceptions.txt [hunk 1]
```diff
@@ -1,31 +1,26 @@
...
 72073: 'Star Trek DS9',
-195831: 'Zane Lamprey\'s Drinking Made Easy',
-76133: 'Poirot', 'Agatha Christie\'s Poirot', 'Agatha Christie Poirot',
+76133: 'Agatha Christie Poirot',
 77444: 'This Old House Program',
 73290: '60 Minutes (US)',
-194751: 'Conan', 'Conan (2010)',
+194751: 'Conan',
 164451: 'Carlos (2010)',
-70726: 'Babylon 5', 'Babylon5',
```
Comment (miigotu) [inline exceptions.txt:28]: This one is required unless we change SickRage's build tests.
https://github.com/SiCKRAGETV/SickRage/blob/master/tests/scene_helpers_tests.py#L103

Comment (mbxIO) [inline exceptions.txt:28]: @miigotu The "Babylon 5" is required? Why is that, since it's identical to TheTVDB name?

Comment (miigotu) [inline exceptions.txt:28]: It is part of the build tests to check that scene exceptions are updated.
Let me test it without in a little bit.

Comment (mbxIO) [inline exceptions.txt:28]: @miigotu sounds good. Let me know when you have, only when you have the time, and I will continue to edit this list. Thanks

**Comment by mbxIO** [comment]
@miigotu Hey man, if you get a chance to look into Babylon 5, I'm off work Thursday and Friday and would love to knock on this updated exceptions list. If you don't have time, no big deal.

**Comment by mbxIO** [comment]
@miigotu Any update on Babylon 5, should I just re-add it for now and move forward?

**Comment by miigotu** [comment]
Just leave it in for now. I don't have time to test it.

**Comment by mbxIO** [comment]
@miigotu Besides Babylon 5, are there any others that you use to test?

**Comment by miigotu** [comment]
From what I see without digging, Carlos 2010, Babylon5, and Babylon 5

**Pull request closed without merging**